### DISCUSSION TITLE
Consistency in object tracking - PLPath vs Stack

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#5402] [General]
In catalog records, the container stack is called PLPath

![image](https://user-images.githubusercontent.com/5725767/148815688-70b24a03-efb3-44ef-8a1f-87ee0e8e1c4a.png)

in the container history it is called Stack

![image](https://user-images.githubusercontent.com/5725767/148815920-eabdc23d-347d-42ac-933b-007ad21802db.png)

Stack in container history will show the last place the thing was before it was moved (same as PLPath) let’s rename to make them consistent.

What does PLPath stand for and would we rather use "Stack" everywhere? Or do neither of these terms do a good job of describing what they represent? If not, what would?

@KyndallH @campmlc @Nicole-Ridgwell-NMMNHS

**Comment by dustymc** [comment]
P[art]L[ocation]Path would not be appropriate for things that don't necessarily involve parts.

**Comment by Jegelewicz** [comment]
How about Storage Location Stack or Storage Location Path?

**Comment by campmlc** [comment]
I like something simpler and clearer, ideally just "Location",  or "Path"
(not very clear) or "Tree" ?

**Comment by KyndallH** [comment]
I vote for either Whereabouts or Location.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I like location or storage location

**Comment by Jegelewicz** [comment]
Consensus seems to be "location"

**Comment by dustymc** [comment]
I think the two "requirements" of this are:

1. Concise; lots of things don't have this information, the label shouldn't take up unnecessary space.
2. Isn't overly confusing with https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecpart_attribute_type; eg "location" is likely to cause problems.

"Tree" doesn't seem quite right - the branches have been stripped for this view. (The source data are certainly a "tree.")

**Comment by campmlc** [comment]
PLocation?

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Storage Path?

**Comment by dustymc** [comment]
>PLocation

Depends what the "P" is for....

>Storage Path?

There's not necessarily anything stored there - this is for containers, whether they contain anything or not and whether that anything involves parts or not.

I'm not sure there are any wrong answers, just some random considerations....

**Comment by campmlc** [comment]
I think Location is good and understandable in this context . . .

**Comment by campmlc** [comment]
Part attributes display differently.

**Comment by Jegelewicz** [comment]
Object tracking group says lets go with location.

**Comment by AJLinn** [comment]
<img width="1298" alt="Screen Shot 2022-01-21 at 3 29 32 PM" src="https://user-images.githubusercontent.com/17605945/150616711-2f72e396-7583-4f2f-80e7-5c39e5a14232.png">

Assuming that doesn't cause some sort of conflict with the Part Attribute called Location.

**Comment by Jegelewicz** [comment]
> Assuming that doesn't cause some sort of conflict with the Part Attribute called Location.

It won't - all we are talking about are labels on the parts table (and in some other places) see https://github.com/ArctosDB/arctos/issues/4239#issue-1098212388

**Comment by dustymc** [comment]
>conflict

For machines: Not at all, it's a different context/thing.

For people: Maybe? It's the same sort of information, sorta, and there's some "same name different thing" conversation somewhere that I can't find right now....

FWIW I'm fine with whatever.

**Comment by Jegelewicz** [comment]
How about **ContainerLocation**? Makes it different from the location attribute and matches up with the "InContainer" label

**Comment by Jegelewicz** [comment]
@campmlc @Nicole-Ridgwell-NMMNHS @KyndallH 

ContainerLocation

?
